frontend/app/assets/javascripts/spree/frontend.js

```diff
@@ -1,18 +1,14 @@
 //= require jquery3
+//= require jquery_ujs
```

Comment:
Now we need it because the new views that will be moved from spree-designs rely on it. See https://github.com/spark-solutions/spree-designs/pull/90 for details